### DISCUSSION TITLE
Harder Google reCaptchas in Chrome 126

### DISCUSSION THREAD

**Discussion by FlowFlaux** [seleniumbase/SeleniumBase#2977] [Q&A] [answered]
The most recent Stable branch update for Chrome made Google's reCaptchas significantly harder, which also triggers other security checks even if the captcha is solved. I doubt rolling back and disabling automatic updates would help, and I can't find any other ways to deal with that. Anyone faced the issue or got any pointers?

**Comment by mdmintz** [comment, marked as answer]
reCAPTCHA is different from other anti-bot services because it may require solving a challenge even if not detecting Selenium or other automation frameworks directly. It does a lot of behavioral analysis, and keeps track of activity coming from IP addresses.

If you can't bypass reCAPTCHA with `solve_captcha()`, then you could try solving the audio challenge with an external repo:
https://github.com/search?q=pydub.AudioSegment.from_mp3+recaptcha+solver+language%3APython&type=code
